dependencies/src/test/java/com/google/cloud/tools/opensource/dependencies/DependencyGraphBuilderTest.java

```diff
@@ -208,6 +208,8 @@ public void testBuildVerboseDependencyGraph_singleArtifact() {
         dependencyGraphBuilder.buildVerboseDependencyGraph(logging);
 
     List<DependencyPath> list = graph.list();
+    // The list should not include provided dependencies
+    Truth.assertThat(list).hasSize(4);
```

Comment:
Instead of asserting the size is 4 and then each one individually, you could use fuzzy truth:

```
assertThat(list)
    .comparingElementsUsing(
        Correspondence.transforming(dp -> dp.getLeaf().getArtifactId(), "is path to artifact with ID")
    .containsExactly("commons-logging", "log4j", "logkit", "avalon-framework")
    .inOrder();
```

Then failure would show a good message with all the problems.

Reply:
That's nice. Updated.